package-lock.json

```diff
@@ -1,9570 +1,8 @@
 {
   "name": "national-npp",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Looks like the lockfileVersion changed in a newer version of npm.  Make sure npm is up to date on your machine and run `npm install` again to get the correct package-lock.json